.gitignore

```diff
@@ -4,3 +4,4 @@ release/
 *.pro.user
 .qmake.stash
 applications/*/Makefile
+result
```

Comment:
```suggestion
result/
```

Reply:
`result` is a symlink, so without the slash is preferable.

Reply:
What does it symlink to?

Reply:
Into the Nix store.
```
$ ls -al result
lrwxr-xr-x 1 siraben 49 Dec 28 17:53 result -> /nix/store/kf4kzzlhafmkixlfqf49mn90dcvc0a3h-oxide
```